src/utils/storage.js

```diff
@@ -13,43 +10,65 @@ const getBlankUserWordList = () => ({
   five: {},
 });
 
+export const getAllUsers = async () => {
+  try {
+    const usersJson = AsyncStorage.getItem('@users').then(JSON.parse);
+    const users = usersJson || {};
+    return users;
+  } catch {
+    return {};
+  }
+};
+
 export const getUser = async () => {
   try {
     const user = await AsyncStorage.getItem('@user');
-    if (user !== null) {
-      return user;
-    }
-    return null;
+    return user;
   } catch (e) {
     return null;
   }
 };
 
+export const addUser = async (user) => {
```

Comment:
Minor note, would probably name this argument something like `username` so it's clear that it's just a string, not a full user object?

Reply:
True, that is a little vague. Also, the functions in that file are admittedly a little confusing, so I went ahead and added docstrings to all of them